examples/app/Login.tsx

```diff
@@ -33,6 +34,21 @@ const Login = () => {
 		[]
 	);
 
+	const logger = useMemo(
+		() => ({
+			info: (message: string, description: string, state: any) => {
```

Comment:
just noting that I don't think implementor need to worry about `state` or such, this is a confusing api

Reply:
nice reference for something currently unavailable (onNext)

Reply:
lets put this out from the sample app, it is just confusing and will raise questions 

this will keep this sample app nice and tidy